SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1152,9 +1158,13 @@ void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*
 
 void USpatialActorChannel::ServerProcessOwnershipChange()
 {
-	if (!IsAuthoritativeServer())
+	SCOPE_CYCLE_COUNTER(STAT_ServerProcessOwnershipChange);
 	{
-		return;
+		SCOPE_CYCLE_COUNTER(STAT_IsAuthoritativeServer);
```

Comment:
This counter will be the same as above, what's the need?

Reply:
It's in a smaller scope, so it catches just the IsAuthoritativeServer. The parent scope includes the ownership changes

Reply:
although I agree that the time for the inner scope should be negligible.

Reply:
Oh, I read this as a counter, not a cycle_counter.